README.md

```diff
@@ -152,16 +152,17 @@ Tady bude ukázka převod slajdů z README.md do HTML a PDF pomocí marp
 
 ## Co když Markdown pro moje účely nestačí?
 
-AsciiDoc je další značkovací jazyk který má větší funkcionalitu a dokáže mnohem víc. Například je schopen plnit následující úkoly:
-- AsciiDoc poskytuje sofistikovaněji strukturu dokumentů
-- Pomocí direktivy `include::` lze přidávat obsah souborů přímo do stávajícího textu
-- Lze využit proměnné pro opakované použití stejného obsahu
-- Lze použit HTML poznámky a upozornění
-- Lze nechat automaticky vygenerovat obsah
+AsciiDoc je dalším značkovacím jazykem, který však disponuje daleko větším množstvím funkcí. Je například schopen plnit následující úkoly:
+
+- AsciiDoc poskytuje sofistikovaněji strukturu dokumentů.
+- Pomocí direktivy `include::` lze přidávat obsah souborů přímo do stávajícího textu.
+- Lze využit proměnné pro opakované použití stejného obsahu.
+- Lze použit HTML poznámky a upozornění.
+- Lze nechat automaticky vygenerovat obsah.
```

Comment:
IBM Style Guide p. 62:

> Periods with lists
> Write lists so that all or none of the items start with complete sentences. If list items comprise
> only complete sentences, include a period after each sentence.
> Example (complete sentences)
> 1. Remove the cover.
> 2. Remove the adapter.
> 3. Replace the adapter.
> 4. Replace the cover.
>
> If list items comprise only sentence fragments, do not include any end punctuation.
> Example (fragments)
> Consider these factors when you design a web page:
> • Background patterns and images
> • Font styles and colors
> • Layout options
>
> If a list contains list items that start with sentence fragments, and one or more of the fragments are
> followed by complete sentences, include periods after all sentence fragments and sentences.
> Example
> Session management can store session-related information in several ways:
> • In application server memory. This storage option is local to the application server and
> cannot be shared with other application servers.
> • In a database.
> • In another WebSphere Application Server instance.

Reply:
To co jsem citoval, vsak rika, ze mas pravdu a tecky tam patri :) Tak to nechavame a dik za opravu :+1:

Reply:
Tady se třeba IBM SG a pravidla pro českou typografii shodují. Pokud je položkou celá věta, pak končí tečkou. :)